Fix failing github actions

It fails sprodically

```
Running App tests...
============================= test session starts ==============================
collected 5 items / 1 error

==================================== ERRORS ====================================
_____________ ERROR collecting src/repositories/test_user_repo.py ______________
ImportError while importing test module '/home/runner/work/uv-workspaces/uv-workspaces/packages/app/src/repositories/test_user_repo.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
src/repositories/test_user_repo.py:5: in <module>
    from testcontainers.mongodb import MongoDbContainer
../../.venv/lib/python3.13/site-packages/testcontainers/mongodb/__init__.py:19: in <module>
    from testcontainers.core.utils import raise_for_deprecated_parameter
E   ImportError: cannot import name 'raise_for_deprecated_parameter' from 'testcontainers.core.utils' (/home/runner/work/uv-workspaces/uv-workspaces/.venv/lib/python3.13/site-packages/testcontainers/core/utils.py)
- generated xml file: /home/runner/work/uv-workspaces/uv-workspaces/test-results/app-junit.xml -
=========================== short test summary info ============================
ERROR src/repositories/test_user_repo.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.96s ===============================
Error: Process completed with exit code 2.
```

--- a/packages/app/pyproject.toml
+++ b/packages/app/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "app"
-version = "0.1.0"
+version = "0.1.1"
 description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.13"
@@ -17,7 +17,7 @@ dependencies = [
 dev = [
     "pytest>=8.3.4",
     "pytest-asyncio>=0.24.0",
-    "testcontainers>=3.7.1",
+    "testcontainers>=4.9.0",
     "testcontainers-mongodb>=0.0.1rc1",
 ]
 

